lib/containers/home.js

```diff
@@ -9,6 +9,11 @@ import ArtworkRail from '../components/home/artwork_rails/artwork_rail'
 import ArtistRail from '../components/home/artist_rails/artist_rail'
 
 class Home extends React.Component {
+  state: {
+    modules: [any],
```

Comment:
hrm, I wonder if this should be `any[]`

or we could define the shape of a module too, not sure whether thats a good idea though, bet there's a lot of variation there